### ISSUE TITLE
End Cap Muon Log Warnings

### ISSUE THREAD

**Issue by aloeliger** [cms-l1t-offline/cmssw#1137]
### What

Right now we are seeing very noisy output from end cap muon modules. The exact log info is:

```
%MSG-w L1T:  L1TMuonEndCapTrackProducer:simEmtfDigis 13-May-2023 11:29:10 CEST  Run: 366727 Event: 121426611
Found error in LCT quality: 0 (allowed range: 1-15).
%MSG
%MSG-w L1T:  L1TMuonEndCapTrackProducer:simEmtfDigis 13-May-2023 11:29:10 CEST  Run: 366727 Event: 121426611
From endcap 2, sector 3, station 2, ring 2, cscid 6. (Note that this LCT may be reported multiple times. See source code for explanations.)
%MSG
%MSG-w L1T:  L1TMuonEndCapTrackProducer:simEmtfDigis 13-May-2023 11:29:10 CEST  Run: 366727 Event: 121426611
Found error in LCT quality: 0 (allowed range: 1-15).
%MSG
```

These warnings originate [here](https://github.com/cms-sw/cmssw/blob/ecf1aaec06678b04066926b8521879a29da4bdc2/L1Trigger/L1TMuonEndCap/src/PrimitiveSelection.cc#L666-L674)

After some discussion I understood this to be a muon unpacker issue with a fix in the works.

@eyigitba I figured I would tag you since we have discussed this.

### Recipe to recreate

2023 data relvals apparently include this right now.

**Comment by eyigitba** [comment]
Hi @aloeliger , this is due to muon shower objects in EMTF DAQ. Currently our unpacker creates an LCT with quality 0 when we get a muon shower bit. This needs to be fixed. I'll submit a PR fixing unpacker for muon shower bits this week.

**Comment by aloeliger** [comment]
@eyigitba Any news on this?

**Comment by eyigitba** [comment]
I'll try to submit the PR today/tomorrow. I decided to include it as a part of a bigger PR involving muon showers in EMTF unpacker. The tests are all good, I need to check just one more thing.

**Comment by eyigitba** [comment]
Hi @aloeliger , 132X PR is here: https://github.com/cms-sw/cmssw/pull/41993

**Comment by aloeliger** [comment]
closed in cms-sw/cmssw#41993
